Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -106,7 +106,8 @@
     <Copy SourceFiles="$(NodeJsExtensionsNpmPackageSourcePath)/config/consumer/.eslintrc.project.json"
           DestinationFiles="$(MSBuildProjectDirectory)/.eslintrc.json"
           Condition="
-            !Exists($(NodeJsExtensionsGlobalESLintConfigurationDirectory))
+            '$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != 'false'
+            AND !Exists($(NodeJsExtensionsGlobalESLintConfigurationDirectory))
```

Comment:
Something went off here.

Reply:
what do you mean?

Reply:
When I checked the diff, it was displayed as something along the lines of 

```xml
    </Target>
         AND !Exists($(NodeJsExtensionsGlobalESLintConfigurationDirectory))
```

Like a bad merge. But not it looks good.